javascript/src/tracing/tracer.ts

```diff
@@ -0,0 +1,41 @@
+import { getLangWatchTracer } from "langwatch";
+import { LangWatchTracer } from "langwatch/observability";
+import { setupObservability } from "langwatch/observability/node";
+import { getEnv } from "../config";
+import { Logger } from "../utils/logger";
+
+const logger = new Logger("tracing.tracer");
+
+/**
+ * Sets up LangWatch observability for scenario testing.
+ * Single responsibility: Initialize tracing infrastructure once per process.
+ */
+const envConfig = getEnv();
+
+export const observabilityHandle = setupObservability({
+  serviceName: "@langwatch/scenario",
```

Comment:
mhmm, not sure about this, maybe something we can allow to be optionally user configurable from env vars/config file?